### DISCUSSION TITLE
Failed to fetch dynamically imported module ( vite storybook 7 )

### DISCUSSION THREAD

**Discussion by jongeir** [storybookjs/storybook#20472] [Help] [answered]
Good day. 
I have been testing out storybook 7 with vite for our latest project and since we use GitHub pages it will need to run there also.

For some reason, I cannot make storybook 7 with vite run correctly.  It works locally but as soon as it is deployed onto the server, everything inside the storybook iframe will not render.

Here is a screenshot from the GitHub pages. This is a new storybook project using storybook 7.0.0-beta.19
Nothing fancy has been added to the project.
![image](https://user-images.githubusercontent.com/8004618/210242592-ae17d79b-eb6d-40fa-b5b1-e3158e05515b.png)
errors from console
![image](https://user-images.githubusercontent.com/8004618/210243097-84eeecfa-d6e1-474e-994c-410e70c186ea.png)

By the way, this works fine in older versions of the storybook that uses Webpack.

**Comment by himerus** [comment]
This should be added as in issue vs a discussion. This is an issue I also just ran across.

**Reply by himerus** [reply, marked as answer]
Created an issue here: https://github.com/storybookjs/storybook/issues/20564

**Comment by sreahard** [comment]
How do you see issues? I am having the same problem and would like to be able to see what if any traction has been made here.
